tests/Kernel/ContactFormRenderTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Drupal\contact\Entity\ContactForm;
+use Drupal\Tests\oe_contact_forms\Kernel\ContactFormTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests that our contact forms renders with ecl markup.
+ */
+class ContactFormRenderTest extends ContactFormTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'ui_patterns',
+    'ui_patterns_library',
+    'oe_theme_helper',
+    'breakpoint',
+    'responsive_image',
+    'image',
+    'oe_theme_contact_forms',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installConfig(['system', 'image', 'responsive_image', 'contact']);
+    $this->container->get('theme_installer')->install(['oe_theme']);
+    $this->container->get('theme_handler')->setDefault('oe_theme');
+    $this->container->set('theme.registry', NULL);
+  }
+
+  /**
+   * Tests that corporate contact form is rendered with the correct ECL markup.
+   */
+  public function testContactForm(): void {
```

Comment:
This does not test that, after submitting the form, the status message is properly themed as we want.

Reply:
thats this part `$messenger->addMessage($full_view);`

Reply:
I mean the full thing: I get on the form, submit it and get that message properly styled. but OK if it's done elsewhere.